torch/csrc/jit/codegen/cuda/expr_evaluator.h

```diff
@@ -26,13 +26,16 @@ class TORCH_CUDA_CU_API ExpressionEvaluator : private OptInConstDispatch {
   //! Bind a concrete value to a named scalar
   void bind(const std::string& name, const IntOrDouble& concrete_value);
 
+  //! Set a concrete value for a parallel dimension
+  void bind(ParallelType pt, Int::ScalarType concrete_value);
```

Comment:
This shouldn't have been pulled in here. Haven't added its definition yet. Will remove in this PR, and follow up in the next.